source/js/main.js

```diff
@@ -269,8 +269,7 @@ let main = {
         return {
           label: child.textContent.trim(),
           href: child.getAttribute(`href`),
-          isActive: !!child.getAttribute(`class`).match(/multipage-link-active/),
-          isHighlighted: !child.getAttribute(`class`).match(/multipage-link-active-no-highlight/)
```

Comment:
Searched through the codebase for `multipage-link-active-no-highlight` and `no-highlight` and didn't get any matched result back. Deleting this line as I think this is obsoleted code.